public/javascripts/handlebars-helpers.js

```diff
@@ -68,4 +68,20 @@ Handlebars.registerHelper('format_height', function(heightInches){
   var inches =  heightInches % 12;
   var americanHeight = feet + "'" + inches + "\"";
   return americanHeight;
-})
\ No newline at end of file
+});
+
+Handlebars.registerHelper('percentageCalc', function(attempt, made){
```

Comment:
these params should have more modular names..dividend and divisor? Since that is what this is really doing and we may reuse it for other stats